master/getting-started/docker/index.md

```diff
@@ -32,15 +32,20 @@ you need to install and use Calico.
 
 ## Tutorials
 
-#### [Simple Policy]({{site.baseurl}}/{{page.version}}/getting-started/docker/tutorials/simple-policy)
+#### [Simple Profile Policy]({{site.baseurl}}/{{page.version}}/getting-started/docker/tutorials/simple-profile-policy)
 
 The above guide demonstrates Calico connectivity cross host, and how to limit
 that connectivity using simple Calico policy.
 
-#### [Advanced Policy]({{site.baseurl}}/{{page.version}}/getting-started/docker/tutorials/advanced-policy)
+#### [Advanced Profile Policy]({{site.baseurl}}/{{page.version}}/getting-started/docker/tutorials/advanced-profile-policy)
 
 The above guide digs deeper into advanced policy configurations for workloads.
```

Comment:
I also think that brief summary could be added to the initial intro for each of the main docs.